### DISCUSSION TITLE
Push messages (tickets) to other third-party apps

### DISCUSSION THREAD

**Discussion by oarnb** [Sellbrite/Sellbrite-API#15] [Q&A]
Is it possible to send messages (tickets) to other third-party apps via integrations?

![Screenshot_3](https://github.com/Sellbrite/Sellbrite-API/assets/138830828/16f75004-d1c1-43aa-b76e-85cec44856d6)

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

There wouldn’t be a way to send or manage messages using the API. Even within the Sellbrite dashboard messages can only be viewed, but not forwarded or replied to.
